nodebuilder/header/config.go

```diff
@@ -28,7 +28,7 @@ func DefaultConfig() Config {
 	}
 }
 
-func (cfg *Config) trustedPeers(bpeers params.Bootstrappers) (infos []peer.AddrInfo, err error) {
+func (cfg *Config) trustedPeers(bpeers p2p.Bootstrappers) (infos []peer.AddrInfo, err error) {
```

Comment:
idea: Similarly to `modp2p.Bootstrappers`, we may want to make `modheader.TrustedPeers` an application- and module-specific type.